ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -239,9 +242,17 @@ const PANEL_MAP = [ALL_DATASET_TYPE, DATASET_TYPE_VARIANT_CALLS, DATASET_TYPE_SV
 const VariantSearchFormContent = React.memo(({ user, displayAnnotationSecondary, datasetTypes }) => (
   <div>
     <ProjectFamiliesField />
-    <VerticalSpacer height={10} />
-    <InlineHeader content="Saved Search:" />
-    {configuredField(SAVED_SEARCH_FIELD)}
+    <Header size="huge" block>
+      <Grid padded="horizontally" relaxed>
+        <Grid.Row>
+          <Grid.Column width={8} verticalAlign="middle">Select a Saved Search (Recommended)</Grid.Column>
+          <SavedSearchColumn width={4} floated="right" textAlign="right">
```

Comment:
There are a total of 16 columns and only 12 are used. Does it look okay?

Reply:
yes grid's handle the padding for unassigned columns properly